scripts/build_cheatsheet.py

```
@@ -0,0 +1,555 @@
+# Render one layer to an inline <svg> string via keymap-drawer.
+# ---------------------------------------------------------------------------
+def draw_layer(annotated_doc: dict, layer: str, combos: bool) -> str:
+    tmp = DOC / f".cheat_{layer}.yaml"
+    tmp.write_text(yaml.safe_dump(annotated_doc, sort_keys=False, allow_unicode=True))
+    cmd = [
+        "uvx", "--from", "keymap-drawer", "keymap", "-c", str(CONF), "draw",
+        "--select-layers", layer, "-j", str(INFO_JSON), str(tmp),
+    ]
+    if not combos:
+        cmd.insert(cmd.index("draw") + 1, "--keys-only")
+    res = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)